straxen/plugins/peaks/peak_basics_som.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import strax
+from straxen.plugins.peaks.peak_basics import PeakBasics
+
+export, __all__ = strax.exporter()
+
+
+@export
+class PeakBasicsSOM(PeakBasics):
```

Comment:
This plugin is used in the SOM context and replaces peak_basics with a new version including the SOM parameters.